### DISCUSSION TITLE
date binding returns "Invalid Date"

### DISCUSSION THREAD

**Discussion by Dave-Perkins-github** [Budibase/budibase#7281] [Help] [answered]
I'm using a BB database and have created a table: Hopper.
Hopper has 2 date columns in it; deliv_date and deliv_by.
I am displaying these dates in a Hopper Table in my form.
They both display with times.  I have tried changing the
displayed format using the Binding in the Value entry
when you click on "Configure Columns".  I have tried
{{ date deliv_by "MM/DD/YY" }}
and
{{ date Hopper.deliv_by "MM/DD/YY" }}

Both times, the form shows "Invalid Date".
Can you tell me what the correct format is?

**Comment by R2bEEaton** [comment, marked as answer]
Hi @Dave-Perkins-github!

The placeholder for the Value entry should show `{{ Value }}`. According to [this](https://docs.budibase.com/docs/table#value), you need to use this for the table value.

Try `{{ date Value "MM/DD/YY" }}` and that should probably work. Hope this helps!

**Comment by Dave-Perkins-github** [comment]
OMG That's perfect!!!!!!  It worked, and I've been trying everything I
could think of.
I would never have figured that out!

THANK YOU!

Is there some way for you to mark it as answered and take credit for it?
Or do I have to do it?

THANK YOU!

**Reply by R2bEEaton** [reply]
No problem! Yes, you have to mark it because I don't have permission. There should be a check mark button to "Mark as answered" on my comment.

**Comment by Dave-Perkins-github** [comment]
Done.  Marked and THANK YOU!
